ROMFS/px4fmu_common/init.d/rc.autostart

```diff
@@ -0,0 +1,318 @@
+#
```

Comment:
I thought I removed it manually. I see that it has an entry in the
.gitignore file but it keeps showing up as a modified file.


--
UAVenture AG
http://www.uaventure.com


On Sun, Aug 2, 2015 at 2:06 PM, Lorenz Meier <notifications@github.com>
wrote:

> In ROMFS/px4fmu_common/init.d/rc.autostart
> <https://github.com/PX4/Firmware/pull/2625#discussion_r36042419>:
>
> > @@ -0,0 +1,318 @@
> > +#
>
> This file is auto-generated and should not be committed. Please rebase and
> remove.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PX4/Firmware/pull/2625/files#r36042419>.
>